include/seqan3/core/algorithm/pipeable_config_element.hpp

```diff
@@ -30,7 +30,7 @@ template <typename derived_t, typename value_t>
 struct pipeable_config_element
 {
     //!\brief The stored config value.
-    value_t value;
+    value_t value{};
```

Comment:
@smehringer told me that if the value is not initialized, the pipeable_config_element sometimes has problems in constexpr contexts. She also told me that the value should be removed when all configuration elements have been added. Maybe she can explain this in more detail.

Reply:
ok

Reply:
Did it solve the issue?

Reply:
@smehringer: No, I thought it would, but it only came at the same time that @eseiler made a change that fixed the problem. I removed the change again to double check and it doesn't help.